index.Rmd

```diff
@@ -4,11 +4,11 @@ author: Suiyuan Wang
 subtitle: A case study in Buffalo city, NY
 ---
 
-# Introduction
+# Introduction 
```

Comment:
I really enjoyed your introduction - it was easy to see what is problem is and what you are going to do in your code.